### DISCUSSION TITLE
Cloudinary support?

### DISCUSSION THREAD

**Discussion by avdean** [sveltia/sveltia-cms#4] [Ideas]
With Forestry ending support, I'm shopping around for a new CMS, and while Decap has been announced, this certainly looks more promising. 

Both Netlify and Decap, support the ability to use Cloudinary as a media library, am I right in thinking the same thing can be done in Sveltia CMS?

Thanks!

**Comment by kyoshino** [comment]
Hey @avdean, thanks for checking out Sveltia CMS! Yes, I plan to support Cloudinary and Uploadcare for better compatibility with Netlify/Decap CMS. Hope it’s no big deal. I’ll let you know once it’s done!

**Reply by avdean** [reply]
Great looking forward to hearing from you once it's done, will definitely try it out!

**Comment by nkitsaini** [comment]
Also want to add that it would be nice if we can have a way to register custom media libraries. I am specifically interested in Cloudflare stream and images, and happy to implement it as a custom media library (even better if officially supported).

**Reply by kyoshino** [reply]
Custom media library support is not planned at this time (see the project README), but I do have plan to add support for Cloudflare.

**Reply by nkitsaini** [reply]
> not planned at this time

Okay

> see the project README

ahh, missed the mention in README. But makes sense.


> I do have plan to add support for Cloudflare

That's great!

**Comment by kyoshino** [comment]
Finally, [v0.113.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.113.0) added Cloudinary support.
